### PR TITLE
Change pillar data for new nginx formula

### PR DESCRIPTION
**Pull request by markbreedlove**
Change some list properties to string properties for the new version of `nginx-formula`. These are mostly `listen` directives, where the formula used to accept a list of the different parts of the line, but now wants a string representing the whole thing.

### CODE REVIEW

**Comment by markbreedlove** [comment]
@blarghmatey I think that's all the necessary changes.

File: pillar/nginx/kibana.sls [hunk 1]
```diff
@@ -53,15 +44,9 @@ nginx:
...
-                    - $host
-                  - proxy_set_header:
-                    - X-Real-IP
-                    - $remote_addr
-                  - proxy_set_header:
-                    - X-Forwarded-For__vault__::secret-operations/global/mitca_ssl_cert>data>value
-                    - $remote_addr
+                  - proxy_set_header: 'Host $host'
+                  - proxy_set_header: 'X-Real-IP $remote_addr'
+                  - proxy_set_header: 'X-Forwarded-For__vault__::secret-operations/global/mitca_ssl_cert>data>value $remote_addr'
```
[the remark this answers is not in the document]
Comment (markbreedlove) [reply pillar/nginx/kibana.sls]: Does commit 8c46406 fix this?

[the remark this answers is not in the document]
Comment (markbreedlove) [reply pillar/nginx/kibana.sls]: I didn't notice your comment above. I will merge this PR and if this vault line doesn't work we can fix it next.

**Comment by blarghmatey** [comment]
:+1:

**Pull request merged**